### PR TITLE
Site Profiler: Align numbers to prevent layout shifts

### PR DESCRIPTION
**Pull request by candy02058912**
Fixes https://github.com/Automattic/dotcom-forge/issues/9290

## Proposed Changes

https://github.com/user-attachments/assets/0ecaee2c-3375-4150-8dec-a16bad945a4a

### CODE REVIEW

**Review by matt-west — APPROVED** [review]
Thanks @candy02058912! This looks good now. 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16888917

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Hi @candy02058912, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `Tested on {{span}}%(testedDate)s{{/span}}. {{button}}Test again{{/button}}`

Thank you in advance!

File: client/hosting/performance/site-performance.tsx [hunk 1]
```diff
@@ -257,6 +257,13 @@ export const SitePerformance = () => {
 							onClick={ retestPage }
 						/>
 					),
+					span: (
+						<span
+							style={ {
+								fontVariantNumeric: 'tabular-nums',
```
Comment (zaguiini) [inline client/hosting/performance/site-performance.tsx:263]: Wow, TIL!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16888917).

**Pull request merged**